persistence-jdbc/core/src/main/resources/reference.conf

```diff
@@ -77,8 +131,8 @@ lagom.persistence.read-side.jdbc {
     # The Slick driver - for backward compatibility only, will be removed in 1.5.0 (@deprecated)
     driver = ${?jdbc-defaults.slick.driver}
 
-    # The JNDI name to use
-    jndiName = ${jdbc-defaults.slick.jndiName}
+    jndiDbName = ${jdbc-defaults.slick.jndiDbName}
```

Comment:
Relates to my suggestion above. 

If we have three separated defaults we can make sure that the read side (journal and user land) is configured with the same `jdniDbName`.

This would be encoded as: 

```
lagom.persistence.read-side.jdbc {
    slick.jndiName=${jdbc-defaults.slick.jndiDbNames.read-journal}
}
```